code/modules/power/solar.dm

```diff
@@ -251,7 +260,7 @@
 		W.play_tool_sound(src, 75)
 		return TRUE
 
-	if(istype(W, /obj/item/stack/sheet/glass) || istype(W, /obj/item/stack/sheet/rglass))
+	if(istype(W, /obj/item/stack/sheet/glass) || istype(W, /obj/item/stack/sheet/rglass) || istype(W, /obj/item/stack/sheet/plasmaglass))
```

Comment:
why do u allow glass, reinforced glass, plasma glass but not reinforced plasma glass

Reply:
short answer: bloat

long answer: i dont want to have to make an entire new subtype just for "this is plasma glass, but stronger"